public/app/core/components/AppChrome/NavLandingPage.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { toIconName, useStyles2 } from '@grafana/ui';
+import { Page } from 'app/core/components/Page/Page';
+import { useNavModel } from 'app/core/hooks/useNavModel';
+
+import { NavLandingPageCard } from './NavLandingPageCard';
+
+interface Props {
+  navId: string;
+}
+
+export function NavLandingPage({ navId }: Props) {
+  const { node } = useNavModel(navId);
+  const styles = useStyles2(getStyles);
+  const directChildren = node.children?.filter((child) => !child.hideFromTabs && !child.children);
+  const nestedChildren = node.children?.filter((child) => child.children && child.children.length);
+
+  return (
+    <Page navId={node.id}>
+      <Page.Contents>
+        <div className={styles.content}>
+          {directChildren && directChildren.length > 0 && (
+            <section className={styles.grid}>
+              {directChildren?.map((child) => (
+                <NavLandingPageCard
+                  key={child.id}
+                  description={child.description}
+                  icon={child.icon ? toIconName(child.icon) : undefined}
+                  text={child.text}
+                  url={child.url ?? ''}
+                />
+              ))}
+            </section>
+          )}
+          {nestedChildren?.map((child) => (
+            <section key={child.id}>
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <h2 className={styles.nestedTitle}>{child.text}</h2>
+              </div>
+              <div className={styles.nestedDescription}>{child.subTitle}</div>
+              <div className={styles.grid}>
+                {child.children?.map((child) => (
+                  <NavLandingPageCard
+                    key={child.id}
+                    description={child.description}
+                    icon={child.icon ? toIconName(child.icon) : undefined}
+                    text={child.text}
+                    url={child.url ?? ''}
+                  />
+                ))}
+              </div>
+            </section>
+          ))}
+        </div>
+      </Page.Contents>
+    </Page>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  content: css({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+  }),
+  grid: css({
+    display: 'grid',
+    gap: theme.spacing(2),
+    gridTemplateColumns: 'repeat(auto-fill, minmax(300px, 1fr))',
+    gridAutoRows: 'minmax(200px, 1fr)',
```

Comment:
this just makes each row a minimum of 200px high?

Reply:
yeah actually this just really needs to be `gridAutoRows: '200px'`, i'll change 👍